test-packages/e2e-tests/package.json

```diff
@@ -57,7 +55,9 @@
     "requires": {
       "db": {
         "kind": "sqlite",
-        "model": "*"
```

Comment:
[q] could you please share the reason?

Reply:
This change was unintended, will revert it.